src/main/java/com/jcabi/dynamo/mock/MkTable.java

```diff
@@ -100,4 +100,16 @@ public Region region() {
     public String name() {
         return this.self;
     }
+
+    /**
+     * Delete item from aws table.
+     * @todo Implement #25
+     * @param attributes Attributes
+     * @throws IOException In case of DynamoDB failure
+     */
```

Comment:
@yegor256 Could you please comment on this? What is correct way to add a puzzle?